### PR TITLE
Remove unnecessary logging

### PR DESCRIPTION
**Pull request by fjvallarino**
@tejohnso please review

### CODE REVIEW

File: installer/network-check.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ module.exports = {
 
     return Promise.all(siteConnections)
       .then((res)=>{
-        log.file(JSON.stringify(res));
+        log.file("Electron network checks passed: " + JSON.stringify(res));
```
Comment (tejohnso) [inline installer/network-check.js]: @fjvallarino isn't this now going to log
>Electron network checks passed: [null, null, ...]

Making it look like nothing passed?

Comment (fjvallarino) [inline installer/network-check.js]: @tejohnso yeah; I only added the "Electron network checks passed: " to know where that log line came from. We don't actually need the result of the computation. In fact, we don't really need the log.

Comment (fjvallarino) [inline installer/network-check.js]: @tejohnso I meant "passed" as in "the network checks were successful"

Comment (tejohnso) [inline installer/network-check.js]: @fjvallarino I think we should remove "passed" and change it to *"Electron network checks:"*
Other than that, :+1:

**Pull request merged**